pkg/util/tls.go

```diff
@@ -30,22 +30,26 @@ func TlsCiphersToIDs(names []string) ([]uint16, error) {
 		// no ciphers specified, use defaults
 		return nil, nil
 	}
-	var ids []uint16
-	var missing []string
 
 	ciphers := tls.CipherSuites()
-	var cipherMap = make(map[string]uint16, len(ciphers))
+
+	cipherMap := make(map[string]uint16, len(ciphers))
 	for _, cipher := range ciphers {
 		cipherMap[cipher.Name] = cipher.ID
 	}
 
+	missing := []string{}
```

Comment:
why not pre-allocating `missing` too? why the linter does not complain 🤔 ?

Reply:
I think the reason is that `ids` is the slice assumed to be more commonly appended to, whereas `missing` is less likely to be appended to, and it's much more difficult to predict the necessary size.

In other words, the happy-path involves `ids` having a final size equal to the preallocated capacity, and an empty `missing` slice.